spec/models/user_spec.rb

```diff
@@ -2,15 +2,12 @@
 
 RSpec.describe User, type: :model do
   subject do
-    User.new(
-      name: 'Codepantherr',
-      photo: 'http://unsplash.com/codepantherr',
-      bio: 'badass dev',
-      posts_counter: 0
-    )
+    User.create(name: 'Prom', bio: 'This is my life',
+                photo: 'https://avatars.githubusercontent.com/u/46839548?v=4',
+                email: 'prom@test.com', password: '123456')
   end
 
-  before { subject.save }
+  # before { subject.save }
```

Comment:
Please remove commented code from the project

Reply:
Thank you for pointing this out @arslanbisharat. Fixed.